COMMUNICATION.md

```diff
@@ -0,0 +1,63 @@
+
+### To be avoided:
+- Don't use optional here. 
+- What’s your suggestion?
+
+### Preferred:
+- **I see you are using an optional as an array of User object in the example above**, wouldn’t it be better to have that property default to an empty array?
+
+- **Okay, so you say using a collectionView here would be over-engineering**, how would you then solve this problem instead?
+
+## Giving advice or tips
+Explain why you think this might be helpful in this particular case. Don’t just prescribe, but try to make your case for a particular approach.
+
+### Avoid:
+- Why not use a CollectionView for this?
+- Read this article:
+
+### Prefer:
+- A collection view might be helpful in this case, **because it would take care of the layout for you. Otherwise you’ll be redoing the same work, implementing your own lightweight collection view instead.**
+- This article might be interesting for you, **as it explains the threats of storing API keys in plaintext.**
+
+## When you feel someone is not communicating clear enough
+If you feel someone’s reaction is not clear enough (i.e. someone’s reaction confuses you, or even agitates you), look at these guidelines together to see what could be improved in the future. If you realise something critical is missing in the guidelines, please add to it.
+
+Try to keep in mind that written communication it’s really easy to misread something or to interpret something out of context. Try not to assume rudeness, ill intent or malice. Sometimes something was poorly phrased or misread. Sometimes we’re switching between contexts too often, and end up being shorter than we should. Try asking for clarification, or if the person could rephrase that, if the meaning is’t clear. 👍 
+
+If following the guidelines don't clear things up, consider a call to solve the issue if the matter is pressing, or wait for the next iOS meeting to continue discussing it.
+
+### Good to keep in mind:
+These are paraphrased or downright copied over from Github’s pull request guideline:
+
+- Familiarise yourself with the context of the issue.
+- If you disagree strongly, consider giving it a few minutes before responding; think before you react. Try to empathise.
+- Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
+- Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
+- Offer ways to simplify or improve code.
+- Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced. Don’t shame, no one started out knowing.
+- Be humble. (“I’m not sure, let’s try…”)
+- Never deal in absolutes. Unless needed. (“We should never store sensitive data in plain text.”)
+- Aim to develop professional skills, group knowledge and product quality, through group critique. That goes both way: try to be open and welcoming to criticism.
```

Comment:
This fixe should go in https://github.com/bakkenbaeck/iOS-playbook/blob/master/GIT_AND_GITHUB.md